assets/js/modules/analytics/index.js

```diff
@@ -119,7 +65,7 @@ export const registerWidgets = ( widgets ) => {
 			priority: 1,
 			wrapWidget: false,
 			modules: [ 'analytics-4' ],
-			isActive: isAnalytics4Active,
+			isActive: true,
```

Comment:
`isActive` should be a function but since it's no longer used, it should just be removed from the widget's registration.

Reply:
@aaemnnosttv Thanks for clarification. Updated